python/services/requestservice/setup.py

```diff
@@ -14,7 +14,7 @@
     author_email='',
     url='',
     license='',
-    install_requires=['websockets', 'dmod-communication>=0.4.1', 'dmod-access>=0.1.1',
+    install_requires=['websockets', 'dmod-communication>=0.4.2', 'dmod-access>=0.2.0',
```

Comment:
Notably [/python/lib/access/dmod/access/_version.py](https://github.com/robertbartel/DMOD/blob/stack_fixes/websockservice/python/lib/access/dmod/access/_version.py) still shows `0.1.1`, so is this an error or are there missing changes in dmod-access, perhaps?